tiled/catalog/adapter.py

```diff
@@ -561,14 +561,8 @@ async def get_distinct(self, metadata, structure_families, specs, counts):
 
         return data
 
-    async def create_node(
-        self,
-        structure_family,
-        metadata,
-        key=None,
-        specs=None,
-        data_sources=None,
-    ):
+    @property
+    def insert(self):
```

Comment:
I have factored out this bit into its own method because it's reused a couple of times throughout the class. Maybe call `_insert`?